packages/portal-proto/src/features/projects/SummaryHeaderLeft.tsx

```diff
@@ -0,0 +1,253 @@
+import { DropdownWithIcon } from "@/components/DropdownWithIcon/DropdownWithIcon";
+import SaveCohortModal from "@/components/Modals/SaveCohortModal";
+import { focusStyles } from "@/utils/index";
+import download from "@/utils/download";
+import { useCoreDispatch } from "@gff/core";
+import { Button, Loader, Tooltip } from "@mantine/core";
+import { useState } from "react";
+import { FiDownload as DownloadIcon } from "react-icons/fi";
+import { ProjectViewProps } from "./ProjectView";
+
+function SummaryHeaderLeft({ projectData }: { projectData: ProjectViewProps }) {
```

Comment:
Maybe `SummaryHeaderControls` is a better name for this component? I spent a while trying to figure out if this was a component that displayed on the left at a breakpoint.

Reply:
Sure